src/lib/components/Progress/Progress.tsx

```diff
@@ -53,27 +53,32 @@ export const Progress: FC<ProgressProps> = ({
   return (
     <>
       <div id={id} aria-label={textLabel} aria-valuenow={progress} role="progressbar" {...props}>
-        {((textLabel && labelText && textLabelPosition === 'outside') ||
-          (progress && labelProgress && progressLabelPosition === 'outside')) && (
+        {(textLabel && labelText && textLabelPosition === 'outside') ||
+        (progress && labelProgress && progressLabelPosition === 'outside') ? (
           <div className={theme.label} data-testid="flowbite-progress-outer-label-container">
             {textLabel && labelText && textLabelPosition === 'outside' && (
               <span data-testid="flowbite-progress-outer-text-label">{textLabel}</span>
             )}
-            {progress && labelProgress && progressLabelPosition === 'outside' && (
+            {labelProgress && progressLabelPosition === 'outside' && (
               <span data-testid="flowbite-progress-outer-progress-label">{progress}%</span>
             )}
           </div>
-        )}
+        ) : null}
+
         <div className={classNames(theme.base, theme.size[size], className)}>
           <div
             style={{ width: `${progress}%` }}
             className={classNames(theme.bar, theme.color[color], theme.size[size])}
           >
             {textLabel && labelText && textLabelPosition === 'inside' && (
-              <span data-testid="flowbite-progress-inner-text-label">{textLabel}</span>
+              <span className="whitespace-nowrap" data-testid="flowbite-progress-inner-text-label">
```

Comment:
This class needs to be moved to the theme.

Reply:
It wasn't actually needed. Removed it.